client/src/modules/dashboard/Chapters/pages/ChapterPage.tsx

```diff
@@ -53,7 +53,10 @@ export const ChapterPage: NextPageWithLayout = () => {
   const router = useRouter();
 
   const clickDelete = async () => {
-    const ok = await confirmDelete({ doubleConfirm: true });
+    const ok = await confirmDelete({
+      body: 'Are you sure you want to delete this chapter? chapter deletion cannot be reversed',
```

Comment:
```suggestion
      body: 'Are you sure you want to delete this chapter? All information related to chapter will be deleted, including events and venues from this chapter. Chapter deletion cannot be reversed.',
```
Slight nitpicking

Reply:
I was worried it would be considered spammy. 👍 In the future, I will create separate PRs when I face similar situation

Reply:
Naah, not spammy at all. There's clear goal behind the other change as well. Couple of smaller, precise PRs are better than single bigger PR, making not very related changes. :slightly_smiling_face: